### DISCUSSION TITLE
Sea Biomes - AcidIsland

### DISCUSSION THREAD

**Discussion by HeyMaark** [BG-Software-LLC/SSB-AcidIslands#25] [Q&A]
# I would like to make a request to add corals at the bottom of the sea like the Vanilla minecraft biome
![image](https://github.com/user-attachments/assets/7fa38438-219f-4771-8f3c-a874fbd14c2e)

# Currently he is like this
![imagem_2024-07-25_041648443](https://github.com/user-attachments/assets/f9bb37d5-bbf2-437e-9d9e-b05b89abef9d)

## The reason for my request would be to have a more suitable look for the newer versions

I apologize if I created the topic in the wrong place.

**Comment by Aides123** [comment]
nice idea

**Comment by HeyMaark** [comment]
up +1

**Comment by Lemvs** [comment]
@OmerBenGera can you move this to the AcidIslands repository?
